Formula/wdc.rb

```diff
@@ -0,0 +1,56 @@
+class Wdc < Formula
+  desc "WebDAV Client provides easy and convenient to work with WebDAV-servers."
+  homepage "https://designerror.github.io/webdav-client-cpp"
+  url "https://github.com/designerror/webdav-client-cpp/archive/v1.0.0.tar.gz"
+  sha256 "649a75a7fe3219dff014bf8d98f593f18d3c17b638753aa78741ee493519413d"
+
+  depends_on "cmake" => :build
+  depends_on "openssl"
+  depends_on "pugixml"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+    (lib+"pkgconfig/libwebdavclient.pc").write pc_file
+  end
+
+  def pc_file; <<-EOS.undent
+    prefix=#{HOMEBREW_PREFIX}
+    exec_prefix=${prefix}
+    libdir=${exec_prefix}/lib
+    includedir=${exec_prefix}/include
+    Name: libwebdavclient
+    Description: Modern and convenient C++ WebDAV Client library
+    Version: 1.0.0
+    Libs: -L${libdir} -lwebdavclient
+    Libs.private: -lpthread -lpugixml -lm -lcurl -lssl -lcrypto
+    Cflags: -I${includedir}
+    EOS
```

Comment:
@designerror Didn't want to block this PR being merged but a `.pc` file should really be generated and installed by your CMake buildsystem. Could you do that for a future version? Thanks!